### DISCUSSION TITLE
Property list

### DISCUSSION THREAD

**Discussion by Japa8597** [microsoft/vscode-cmake-tools#4192] [Ideas]
>b<"featureasystemon">b<

>c<"brach10linesconectedfirst">c<

>d<"wordsareasdiferentscolorson">d<

>e<"linktoblaindcomands">e<

>f<"adresssheethtplaccess">f<

>g<"desplegatemenulinkmenumain">g<

>h<" exceljoincreatesizemenu">h<

(>h1 -h5 : taman̈o de letras del parrafo )

>i<"marktransparentbackgroundpageoninsert">i<

>j<"attachcomand[example-[   ])designmovementjournayonesitetootheropccionmenu">j<

>k<"attach>a<,>b<,>c<,>d<,>e<,>f<,>g<,>h<,>i<,>j<">k<

>m<" urldirecttootherwindowhentouchopccionselected>>href "m"

**Comment by gcampbell-msft** [comment]
Could you provide more context in a new discussion post? I'm not sure what this is intended to mean. Thanks!
